code/game/dna/dna_modifier.dm

```diff
@@ -50,8 +50,8 @@
 	var/obj/item/reagent_container/glass/beaker = null
 	var/opened = 0
 
-/obj/machinery/dna_scannernew/New()
```

Comment:
pointless because we dont have any dna scanners or ways for players to construct them